### PR TITLE
RabbitMQ to ServiceControl transport adapter post-review fixes

### PR DESCRIPTION
**Pull request by MarcinHoppe**

### CODE REVIEW

File: samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/Adapter/Program.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ static void Main()
             customization: transport =>
             {
                 transport.ConnectionString("host=localhost");
+                transport.DelayedDelivery().DisableTimeoutManager();
```
Comment (SimonCropp) [inline samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/Adapter/Program.cs]: docs snippets tend to avoid the fluent api variants eg https://docs.particular.net/transports/rabbitmq/delayed-delivery#backwards-compatibility-disabling-the-timeout-manager

File: samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/RabbitMQTransportAdapter.sln.DotSettings [hunk 2]
```diff
@@ -1,6 +0,0 @@
-<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```
Comment (SimonCropp) [inline samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/RabbitMQTransportAdapter.sln.DotSettings]: why is this deleted?

File: samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/Sales/Program.cs [hunk 3]
```diff
@@ -48,6 +50,7 @@ static void Main()
         recoverability.Delayed(delayed => delayed.NumberOfRetries(0));
         recoverability.DisableLegacyRetriesSatellite();
 
+        transport.Routing().RouteToEndpoint(typeof(ShipOrder).Assembly, "Samples.ServiceControl.RabbitMQAdapter.Shipping");
```
Comment (SimonCropp) [inline samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/Sales/Program.cs]: assign a local variable for the result of routing

File: samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/RabbitMQTransportAdapter.sln.DotSettings (hunk 2 above)
Comment (mauroservienti) [inline samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/RabbitMQTransportAdapter.sln.DotSettings]: Not using R# I have no idea. @MarcinHoppe ?

**Comment by mauroservienti** [comment]
Addressed @SimonCropp's comments

File: samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/RabbitMQTransportAdapter.sln.DotSettings (hunk 2 above)
Comment (bording) [inline samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/RabbitMQTransportAdapter.sln.DotSettings]: Do we typically include R# settings files in samples? Seems odd to me that we would.

Comment (SimonCropp) [inline samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/RabbitMQTransportAdapter.sln.DotSettings]: Yes we use r# settings in samples since it allows us to keep the code consistent

Comment (bording) [inline samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/RabbitMQTransportAdapter.sln.DotSettings]: Fair enough. Guess I just haven't noticed them before.

**Comment by mauroservienti** [comment]
@MarcinHoppe once merged we can take care of #3037
